docs/misc/syslog.md

```diff
@@ -0,0 +1,25 @@
+Centralized logging with syslog
```

Comment:
Can we add a link to this doc into the k8s-cluster and slurm-cluster docs?

Reply:
Done!